### PR TITLE
Easy admin bundle

### PR DESCRIPTION
**Pull request by pueppiblue**

### CODE REVIEW

File: app/DoctrineMigrations/Version20160803183748.php [hunk 1]
```diff
@@ -18,11 +18,11 @@ public function up(Schema $schema)
         // this up() migration is auto-generated, please modify it to your needs
         $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
 
-        $this->addSql('CREATE TABLE clm_characters (id INT AUTO_INCREMENT NOT NULL, user_id INT DEFAULT NULL, character_name VARCHAR(190) NOT NULL, clm_class VARCHAR(190) NOT NULL, preferred_set VARCHAR(190) DEFAULT NULL, INDEX IDX_3156DDECA76ED395 (user_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE clm_character (id INT AUTO_INCREMENT NOT NULL, user_id INT DEFAULT NULL, character_name VARCHAR(190) NOT NULL, clm_class VARCHAR(190) NOT NULL, preferred_set VARCHAR(190) DEFAULT NULL, INDEX IDX_3156DDECA76ED395 (user_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB');
```
Comment (robertfausk) [inline app/DoctrineMigrations/Version20160803183748.php]: Create new migrations instead of editing old ones.

File: app/config/config.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
     - { resource: services.yml }
+    - { resource: config_eadmin.yml }
```
Comment (robertfausk) [inline app/config/config.yml]: Wording ```config_eadmin.yml``` confuses cause it's similar to ```config_dev.yml```.
I think ```eadmin.yml``` would be better.

File: src/AppBundle/Entity/ClmAccount.php [hunk 3]
```diff
@@ -50,14 +50,32 @@ class ClmAccount
      * User constructor.
      * @param string $name
      */
-    public function __construct($name)
+    public function __construct($name = '')
```
Comment (robertfausk) [inline src/AppBundle/Entity/ClmAccount.php]: Should it be possible to have an empty name?

File: src/AppBundle/Entity/ClmCharacter.php [hunk 4]
```diff
@@ -38,13 +38,21 @@ class ClmCharacter
      * @param $name
      * @param $clmClass
      */
-    public function __construct($name, $clmClass = '')
+    public function __construct($name = '', $clmClass = '')
```
Comment (robertfausk) [inline src/AppBundle/Entity/ClmCharacter.php]: Should it be possible to have an empty name and an empty clmClass?

File: src/AppBundle/Entity/ClmItem.php [hunk 5]
```diff
@@ -35,17 +35,24 @@ class ClmItem
...
      */
-    private $isCash;
+    private $isCash = false;
 
     /**
      * Item constructor.
      * @param string $name
      */
-    public function __construct($name)
+    public function __construct($name = '')
```
Comment (robertfausk) [inline src/AppBundle/Entity/ClmItem.php]: Should it be possible to have an empty name?

File: app/DoctrineMigrations/Version20160803183748.php (hunk 1 above)
Comment (pueppiblue) [inline app/DoctrineMigrations/Version20160803183748.php]: Fox complex changes more fail safe. But in this case the ROI was not good enough for me to put up an entire migration for a simple character difference.

File: src/AppBundle/Entity/ClmAccount.php (hunk 3 above)
Comment (pueppiblue) [inline src/AppBundle/Entity/ClmAccount.php]: EAdmin instantiates the object upon rendering the "new" form and thus cannot provide a name yet.
I could override the createNewClmAccountEntity() , but i would have to provide a dummy name there which then will be overwritten by the form input. Not very elegant either.
On the other hand $name is part of the primary key and thus required to persist the object to the database. I think its just the way easyadmin handles the entities.
Both ways will work. I'm eager for other suggestions from your side.

Comment (robertfausk) [inline src/AppBundle/Entity/ClmAccount.php]: The attribute ```accountName``` isn't part of id (and also shouldn't be) but it's not nullable so it's true that it have to be set to persist the object into database.

I would prefer to omit setting params in constructor as long as we do not use DDD approach or have another good reason to do so.

**Pull request merged**